exercises/grep/grep_test.py

```diff
@@ -36,17 +37,55 @@
 Restore us, and regain the blissful Seat,
 Sing Heav'nly Muse, that on the secret top
 Of Oreb, or of Sinai, didst inspire
-That Shepherd, who first taught the chosen Seed
-'''
+That Shepherd, who first taught the chosen Seed'''
+FILENAMES = [
+    ILIADFILENAME,
+    MIDSUMMERNIGHTFILENAME,
+    PARADISELOSTFILENAME,
+]
+builtins.FILES = {}
```

Comment:
Does this need to be `builtins.FILES` rather than just `FILES`?

Reply:
When I tested with just `FILES`, an empty dictionary was being checked for file contents when the new `open()` was called from the solution file instead of the populated one.

Reply:
I'm not sure what other issue was at play when I tested it then. I will try making the suggested change and see how Travis handles it. If it passes I have no argument against it.

Reply:
It passed.